Userland/Services/Taskbar/ClockWidget.cpp

```diff
@@ -21,9 +23,20 @@ ClockWidget::ClockWidget()
     set_frame_thickness(1);
 
     m_time_width = font().width("22:22:22");
-
     set_fixed_size(m_time_width + 20, 21);
 
+    auto location = Config::read_string("Taskbar", "Appearance", "Location");
```

Comment:
I don't really like that we're reading a config value directly in a Widget, and in a constructor too. What happens if this read fails? What if we try to read this value after we have pledged to not touch any filesystem paths?